command/snapshot/inspect/snapshot_inspect.go

```diff
@@ -29,10 +29,21 @@ type cmd struct {
 	flags  *flag.FlagSet
 	help   string
 	format string
+
+	// flags
+	detailed bool
```

Comment:
Per discussion on flags:
Totally chill with leaving 3 flags, but removing some ambiguity in the naming around what these are would be great. Changing `detailed` to something like `kvInspect` or `kvDetailed` or something fun like that :)